uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/util/JGitUtil.java

```diff
@@ -663,15 +685,46 @@ private static PersonIdent buildPersonIdent( final Git git,
     /**
      * Creates an in-memory index of the issue change.
      */
+    public static DirCache createTemporaryIndexForContent( final Git git,
```

Comment:
we already have 4 createTemporaryIndex; instead of creating a new one.. isn't it possible to improve existing or refactor them to get a better result?